src/cpp/shared/game/game.cpp

```diff
@@ -233,6 +230,13 @@ void game::SetRandomHeroArmies(int heroIdx, int isAI) {
   }
 }
 
+void game::ClearArmy(class armyGroup * heroArmy) {
```

Comment:
This actually makes more sense to put in the armyGroup class.

So, then instead of writing "this->ClearArmy(heroArmy)", you'd write heroArmy->Clear();

Reply:
Noted. Covering this now. I was thinking about that earlier, but I guess I forgot about it.